gordian/gordian.py

```diff
@@ -84,6 +84,14 @@ def get_basic_parser():
         dest='force_changelog',
         help='Fail if changelog does not exist or cannot be parsed'
     )
+    parser.add_argument(
+        '-l','--labels',
+        required=False,
+        default=None,
```

Comment:
Wha if the default is set to `[]` so that you don't need the check on line 160?

Reply:
Makes more sense, I'll do that instead.

Reply:
Fixed in: https://github.com/argoproj-labs/gordian/pull/22/commits/4a941780056e655665cc8e1ac18ab9b1f10eda67 👍